run/test/specs/utils/check_colour.ts

```diff
@@ -23,3 +24,28 @@ export async function parseDataImage(base64: string) {
   // console.info("Middle x:", middleX, "middleY:", middleY, "width:", width);
   return pixelColor;
 }
+
+// Function to compare two colors within a specified CIEDE2000 tolerance
+export function compareColors(
+  hex1: string,
+  hex2: string,
+  tolerance: number = 2.3 // looks-same default value which is "enough for most cases"
```

Comment:
don't duplicate values that don't need to be.
If `looks-same` changes their default in another release, we will be stuck with that one. I assume we don't want to, so make `tolerance ?: number` and deal with the type issues below by checking if tolerance is defined or not.
Or remove tolerance altogether as we don't seem to be using it.